### ISSUE TITLE
Compute requirement

### ISSUE THREAD

**Issue by amandpkr** [M-E-AGI-Lab/Muddit#3]
Hi Authors,

Thank you for your awesome work . Can you please kindly let us know. What is the compute required to the train the model ?

**Comment by viiika** [comment]
Thanks for your attention.
Due to the strong Meissonic prior, we trained Muddit on 8 H100 GPUs for 7 days, 6 days for pre-training and 1 day for supervised fine-tuning. We use a learning rate of 1e-4 with no warm-up and a batch size of 1024. The text-loss weight was set to 0.6.
